src/Uno.UI.sln

```diff
@@ -3193,186 +3193,177 @@ Global
 		{21206058-D5EF-466D-8E4D-AC62357FB479}.Release|x64.Build.0 = Release|Any CPU
```

Comment:
Needs to be merged?

Reply:
Yep because I removed useless platforms in projects.